examples/db-templates/mysql-ephemeral-template.json

```diff
@@ -267,11 +267,11 @@
             "value": "sampledb"
         },
         {
-            "description": "Version of MySQL image to be used (5.7, or latest).",
+            "description": "Version of MySQL image to be used (8.0, or latest).",
             "displayName": "Version of MySQL Image",
             "name": "MYSQL_VERSION",
             "required": true,
-            "value": "5.7"
+            "value": "8.0"
```

Comment:
there is a known issue with running our templates with mysql:8 ... the SCL team is suppose to be working it

in the interim, you should keep this at 5.7

Reply:
done!

Reply:
fyi ... tracking item for  SCL issue is https://bugzilla.redhat.com/show_bug.cgi?id=1793116